### ISSUE TITLE
Value text overlap on StackedBarChart

### ISSUE THREAD

**Issue by petrusgomes** [PhilJay/MPAndroidChart#869]
Hello,

I'm trying to build a stacked horizontal bar chart, following the StackedBarActivity example, but the values texts are showing like this (first chart is buggy, should show 288 & 12 separated):

![device-2015-07-15-160016](https://cloud.githubusercontent.com/assets/3369552/8707234/fbd9bb8e-2b0a-11e5-85b2-ac5b9b014e10.png)

The problem seems to be the large scale from 288 to 12.
Any ideas of how to fix this?

I'm using:

chart.setDrawValueAboveBar(false);

ArrayList<BarEntry> yVals = new ArrayList<BarEntry>();
yVals.add(new BarEntry(new float[] {12, 288}, 0));

Thanks

**Comment by PhilJay** [comment]
Hmmm yea there is not much I can do about this, you will have to modify the library and apply some kind of fix I suppose

**Comment by Im-PJ** [comment]
@petrusgomes Can you share some code about how you implemented the horizontal stacked bar chart? I'm stuck here pls.

**Comment by petrusgomes** [comment]
@Im-PJ I followed the StackedBarActivity example

https://github.com/PhilJay/MPAndroidChart/blob/master/MPChartExample/src/com/xxmassdeveloper/mpchartexample/StackedBarActivity.java

But used HorizontalBarChart instead of BarChart. As I remember, you might have to change the data arrays to match the Y and X changes. 
I ended not using this library in our app, so I don't have the code to put here. Sorry
